app/views/posts/new.html.erb

```diff
@@ -0,0 +1,5 @@
+<%= content_for :title, 'Add new post | ' %>
+
+<h1 class='form_title'>New post</h1>
+
+<%= render 'shared/form', model: @post url: user_posts_path(@user), fields: [:title, :text] %>
```

Comment:
- [ ] When I try to access the form page for the form, I get this error:

![image](https://github.com/Kgomotso196/Blog/assets/77106962/6d1fb5e0-bb4c-4ba9-b0be-fcddb010a2b0)

---

You have a syntax error in this view file. You forgot to use a **_comma (,)_** after `mode: @post`. Kindly use a comma. 👍😎

Reply:
Thank you for your constructive feedback, working on it @Gambit142